virttest/utils_test/libvirt.py

```diff
@@ -3760,3 +3761,354 @@ def check_dumpxml(vm, content, err_ignore=False):
                 raise exceptions.TestFail("Expected '%s' was not found in "
                                           "%s's xml" % (content, vm.name))
     return True
+
+
+#cgroup related functions
+class CgroupTest(object):
+
+    """Class for cgroup related test"""
+
+    __vm_pid = ""
+    __virsh_blkiotune_output_mapping = {"weight": "weight",
```

Comment:
maybe move them ahead to this class, let them as static variables, in this way, other file can easily refer to them if necessary.

Reply:
Done, now they're public consts